doc/modules/preprocessing.rst

```diff
@@ -554,6 +554,10 @@ features into discrete values. Certain datasets with continuous features
 may benefit from discretization, because discretization can transform the dataset
 of continuous attributes to one with only nominal attributes.
 
+One-hot encoded discretized features can make a model more expressive, while
+maintaining interpretability. Pre-processing with a discretizer can introduce
```

Comment:
The second sentence seems a bit detached to me. When I read it, adding something like "However, bare in mind", or "But please note that" or something at the beginning of it, kinda shows that you're warning the user (if that's the intention of the second sentence).

Reply:
I guess it's acceptable because it's just an example. Let's not focus on these minor issues and thanks @adrinjalali for the review.

Reply:
I meant it as "for instance". I can add that. Thanks for the review